plugins/TopPairElectronPlusJetsSelectionFilter.cc

```diff
@@ -533,6 +546,24 @@ void TopPairElectronPlusJetsSelectionFilter::cleanedJets() {
 	}
 }
 
+pat::JetCollection TopPairElectronPlusJetsSelectionFilter::applyNewJec( pat::JetCollection jets, edm::Event& iEvent, const edm::EventSetup& iSetup ) {
+	pat::JetCollection newJets;
+
+	for (unsigned index = 0; index < jets.size(); ++index) {
+		const pat::Jet jet = jets.at(index);
+		float JEC = getJECForJet( jet, iEvent, iSetup);
+		pat::Jet newJet( jet );
+		newJet.setP4( jet.p4() * JEC );
```

Comment:
Found a bug here.  It should be:

newJet.setP4( jet.correctedJet("Uncorrected").p4() * JEC );